### ISSUE TITLE
Scrobbling YouTube videos with chapters broken

### ISSUE THREAD

**Issue by hans-juergen** [web-scrobbler/web-scrobbler#3823]
I have seen a report on Reddit that scrobbling chapters in a video might be broken and splits a single song into chapters now:

https://www.reddit.com/r/lastfm/comments/13vxonb/web_scrobbler_issue/

Maybe automatic chapter scrobbling should be an option that you can disable in the settings?

**Comment by yayuyokitano** [comment]
Thank you for reporting!
This is the currently intended behavior so not technically a bug, but it seems like adding some sort of toggle could be a good idea.
Changing category to feature.

**Comment by hans-juergen** [comment]
Right, some Web Scrobbler users do not seem to know about this behaviour on video chapters, so a toggle could make them aware of it, probably in the YouTube settings. I will add it in the official YouTube FAQ on the support forum as well then:

https://support.last.fm/t/youtube-and-youtube-music-scrobbling/173

**Comment by andreymal** [comment]
An example of a single song split into chapters https://www.youtube.com/watch?v=yfQlZqg9eGg

**Comment by deeplyarnab** [comment]
I'd suggest adding it as a drop-down with multiple choices. Reason being you can have a 3rd option where it's disabled for any track below 6 mins or so, thus songs with chapters indicating intro, chorus, drop, etc don't get recognized as individual tracks for reasonable length music videos, Whereas if it's reasonable to assume any 30min (or >10min) video is podcast or compilation.
Thus, 3 options being enable (always), smart, disabled.
Technically it may be possible to even blacklist the words for short tracks, like if "intro" etc is detected for <6min video then don't scrobble that & look for title via title.
